sms/v1/openapi.yaml

```diff
@@ -1049,17 +1086,17 @@ components:
           description: "The received message as a UTF-8 encoded string. Note: Missing parts of a concat message will be marked with <...> in the message."
           example: "This is an example, the data is garbage."
         originMCC:
-          type: string
+          type: number
           description: Representative MCC (Mobile Country Code) of the originating network
           example: 260
         originMNC:
-          type: string
+          type: number
```

Comment:
Could not test it. Is this correct?

Reply:
Nope according to the code, those values are indeed strings. So the `example` must have double quotes and the type should remain `string`

Reply:
OK, I've replaced d4629bd39ff1c6f8e56563f3cf66749066d57de7 with c173ae01379afe3b080a12afc1c0adefd75ff6ac.

Reply:
Oh, sorry, I hadn't changed originMCC. So, I've replaced f69d724d10c1c9d2ed6e42ed003afc56cfebafab with ed251df9e97f2ac5b77d337b24b26bebbb9a4a68 too.